mw-server/db/query/ways.query.sql

```diff
@@ -240,3 +240,47 @@ RETURNING *,
 -- name: DeleteWay :exec
 DELETE FROM ways
 WHERE uuid = $1;
+
+-- name: GetWayDetailsById :many
+SELECT
+    w.uuid AS way_uuid,
+    w.name AS way_name,
+    w.goal_description AS way_goal_description,
+    w.updated_at AS way_updated_at,
+    w.created_at AS way_created_at,
+    w.estimation_time AS way_estimation_time,
+    w.is_completed AS way_is_completed,
+    w.copied_from_way_uuid AS way_copied_from_way_uuid,
+    w.is_private AS way_is_private,
+    ARRAY(
+            SELECT composite_ways.child_uuid
+            FROM composite_ways
+            WHERE composite_ways.parent_uuid = w.uuid
+    )::VARCHAR[] AS children_uuids,
```

Comment:
After last updateds we dont need children_uuids anymore, let's use childrenAmount

Reply:
done